src/dns_private.tf

```diff
@@ -41,4 +41,18 @@ resource "azurerm_private_dns_a_record" "private_dns_a_record_management" {
   resource_group_name = azurerm_resource_group.rg_vnet.name
   ttl                 = 300
   records             = module.apim.*.private_ip_addresses[0]
+}
+
+# Private DNS Zone for Postgres Databases
+
+resource "azurerm_private_dns_zone" "postgres" {
+  name                = "private.postgres.database.azure.com"
+  resource_group_name = azurerm_resource_group.rg_vnet.name
+}
+
+resource "azurerm_private_dns_zone_virtual_network_link" "postgres" {
```

Comment:
better to use convention dns_zone_vnet_name

So it will be postgres_vnet

Reply:
OK i'll fix